File: CocoaPods--ObjectiveC/IOS-Sample/IOS-Sample/ViewController.m
//
//  ViewController.m
//  IOS-Sample
//
//  Created by Pernic on 20/11/2017.
//  Copyright © 2017 Pernic. All rights reserved.
//

#import "ViewController.h"
#import <Vectaury/Vectaury.h>

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    [Vectaury requestAuthorization];
    [Vectaury startLocation];
    [Vectaury setlanguage:@"fr"];
    // Do any additional setup after loading the view, typically from a nib.
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: CocoaPods--Swift/IOS-Sample/IOS-Sample/ViewController.swift
//
//  ViewController.swift
//  IOS-Sample
//
//  Created by Pernic on 17/11/2017.
//  Copyright © 2017 Pernic. All rights reserved.
//

import UIKit
import Vectaury

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        
        Vectaury.requestAuthorization()
        Vectaury.setlanguage("fr")
        Vectaury.startLocation()
        
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

